### PR TITLE
Fix path in script runner

### LINKED ISSUE #4565: Storing rewards not working for polkadot
**Describe the bug**

Rewards are not stored in staging and production. Reward script works well when running standalone so should be a bug in  scriptRunner.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

### PR DESCRIPTION
**Pull request by mariopino**
Closes #4565 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: scriptRunner/src/polkadotrewards/getPolkadotRewards.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-        )
-        rewardsScript.on('close', function (code) {
-            process.stdout.write(
-                'rewardsScript finished with code ' + code + '\n'
-            )
+  return new Promise((resolve, reject) => {
+    const rewardsScript = spawn(
+      'node',
+      [
+        '../api/scripts/getOldPolkadotRewardEras.js', // the path is in context of the root directory
```
Comment (faboweb) [inline scriptRunner/src/polkadotrewards/getPolkadotRewards.js:8]: the path didn't actually change

File: api/scripts/getOldPolkadotRewardEras.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const database = require('../lib/database')
 const config = require('../config')
 const { ApiPromise, WsProvider } = require('@polkadot/api')
 const _ = require('lodash')
-const BN = require('bignumber.js')
+const BN = require('bn.js')
```
Comment (Bitcoinera) [inline api/scripts/getOldPolkadotRewardEras.js:5]: :eyes:

Comment (Bitcoinera) [reply api/scripts/getOldPolkadotRewardEras.js:5]: What is the difference between these two?

File: api/scripts/getOldPolkadotRewardEras.js [hunk 3]
```diff
@@ -109,22 +109,27 @@ function parseRewards(
...
-          ? BN(allValidatorPreferences[validatorId].commission)
+        const validatorCommission = allValidatorPreferencesUnwrapped[validatorId]
+          ? new BN(allValidatorPreferencesUnwrapped[validatorId].commission)
           : ZERO
-        const available = BN(eraReward)
-          .times(validatorPoints)
-          .dividedBy(eraPoints)
+        const available = new BN(eraReward)
+          .mul(validatorPoints)
+          .div(eraPoints)
```
Comment (Bitcoinera) [inline api/scripts/getOldPolkadotRewardEras.js:127]: oh, I see. Just a different library for doing the same

File: api/scripts/getOldPolkadotRewardEras.js [hunk 4]
```diff
@@ -222,62 +230,19 @@ async function loadEraData(missingEras, api) {
...
-      _.toPairs(storedEraRewards)
-        .filter(([era]) => era >= minDesiredEra)
-        .concat(_.toPairs(eraRewards))
-    ),
-    storedExposures.filter(({ era }) => era >= minDesiredEra).concat(exposures)
-  ]
-
-  return newEraData
-}
-
```
Comment (Bitcoinera) [inline api/scripts/getOldPolkadotRewardEras.js:253]: :eyes:

Comment (Bitcoinera) [reply api/scripts/getOldPolkadotRewardEras.js:253]: How polkadot works changed once more, no?

File: api/scripts/getOldPolkadotRewardEras.js (hunk 3 above)
Comment (faboweb) [reply api/scripts/getOldPolkadotRewardEras.js:127]: yep, but for passing the eras back to the api library we need to use that library

Comment (Bitcoinera) [reply api/scripts/getOldPolkadotRewardEras.js:127]: then why not just use `bignumber`? What am I missing?

Comment (faboweb) [reply api/scripts/getOldPolkadotRewardEras.js:127]: because I didn't think we should have 2 separate libraries that do the same in this script

**Pull request merged**